installing/installing_gcp/preparing-to-install-on-gcp.adoc

```diff
@@ -0,0 +1,48 @@
+
+[id="requirements-for-installing-ocp-on-gcp"]
+== Requirements for installing {product-title} on GCP
+
+Before installing {product-title} on Google Cloud Platform (GCP), you must create a service account and configure a GCP project. See xref:../../installing/installing_gcp/installing-gcp-account.adoc#installation-gcp-project_installing-gcp-account[Configuring a GCP Project] for details about creating a project, enabling API services, configuring DNS, GCP account limits, and supported GCP regions.
+
+If the cloud identity and access management (IAM) APIs are not accessible in your environment, or if you do not want to store an administrator-level credential secret in the `kube-system` namespace, see xref:../../installing/installing_gcp/manually-creating-iam-gcp.adoc#manually-creating-iam-gcp[Manually creating IAM for GCP] for other options.
+
+[id="choosing-an-method-to-install-ocp-on-gcp"]
+== Choosing a method to install {product-title} on GCP
+
+You can install {product-title} on GCP by using one of the following deployment methods.
+
+* **xref:../../installing/installing_gcp/installing-gcp-default.adoc#installing-gcp-default[Installing a cluster quickly on GCP]**: You can install {product-title} on GCP infrastructure that is provisioned by the {product-title} installation program. You can install a cluster quickly by using the default configuration options.
+
+* **xref:../../installing/installing_gcp/installing-gcp-customizations.adoc#installing-gcp-customizations[Installing a customized cluster on GCP]**: You can install a customized cluster on GCP infrastructure that the installation program provisions. The installation program allows for some customization to be applied at the installation stage. Many other customization options are available xref:../../post_installation_configuration/cluster-tasks.adoc#post-install-cluster-tasks[post-installation].
+
+* **xref:../../installing/installing_gcp/installing-gcp-network-customizations.adoc#installing-gcp-network-customizations[Installing a cluster on GCP with network customizations]**: You can customize your {product-title} network configuration during installation, so that your cluster can coexist with your existing IP address allocations and adhere to your network requirements.
+
+* **xref:../../installing/installing_gcp/installing-restricted-networks-gcp.adoc#installing-restricted-networks-gcp[Installing a cluster on GCP by using an internal mirror]**: You can install {product-title} on GCP infrastructure that you provide by using an internal mirror of the installation release content. You can use this method to install a cluster that does not require an active Internet connection to obtain the software components. While you can install {product-title} by using the mirrored content, your cluster still requires Internet access to use the GCP APIs.
```

Comment:
Should this bullet say _Installing a cluster on GCP in a restricted network with user-provisioned infrastructure_ to match the linked topic? It made me think the link was bad when I first clicked. I see it both ways in the docs.

For ex: _[Installing a cluster on AWS by using an internal mirror](https://docs.openshift.com/container-platform/4.8/installing/installing_aws/preparing-to-install-on-aws.html#choosing-an-method-to-install-ocp-on-aws)_ goes to _Installing a cluster on AWS in a restricted network with user-provisioned infrastructure_. But _[Installing a user-provisioned bare metal cluster on a restricted network](https://docs.openshift.com/container-platform/4.8/installing/installing_bare_metal/preparing-to-install-on-bare-metal.html#choosing-a-method-to-install-ocp-on-bare-metal)_ links to _Installing a user-provisioned bare metal cluster on a restricted network_. The latter makes more sense to me.

Reply:
Agreed, thank you sir! Changed per  your observation!